src/App.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import {
+  BrowserRouter, Routes, Route,
+} from 'react-router-dom';
+import './App.css';
+import Home from './components/Home';
+import BookCollection from './components/BookCollection';
+import NotFound from './components/NotFound';
+import Navbar from './components/Navbar';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <BrowserRouter>
+          <Navbar />
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="*" element={<NotFound />} />
+            <Route path="/categories" element={<BookCollection />} />
```

Comment:
- [ ] You should update this link with a "Categories" component instead 💯 Kindly note that the **BookCollection** component is not the same as **Categories** component. Your **Categories** component is missing. Please address that 🙏🏽 The **BookCollection** is a component within the Book component 📖
Kindly follow [this visual](https://pedantic-beaver-cb923b.netlify.app/) to have an idea of how the components are organized.

Reply:
My component BookCollection is actually my component categories

Reply:
- [ ] In my opinion, i think your `BookCollection` component is actually the `BookList` component which renders a list of books using the `<Book>` component, let us kindly create the `Categories` component as mentioned in the previous code review